src/liboslexec/llvm_ops.cpp

```diff
@@ -91,7 +91,15 @@ examples), as you are just coding in C++, but there are some rules:
 */
 
 
+// Some gcc versions on some platforms seem to have max_align_t missing from
```

Comment:
It might also be broken only in particular gcc releases. In any case, that's just a safeguard to make sure it's defined properly on those platforms where somehow it doesn't pick it up. Because it's in a private namespace before any headers are included, it will be subsumed by the one in std, on systems that provide it properly.

This is all a "C++11 growing pains" problem exacerbated by the fact that we're stuck on such an old clang.

Working on that.